ddtrace/contrib/internal/yaaredis/patch.py

```diff
@@ -32,6 +34,8 @@ def get_version():
 
 def patch():
     """Patch the instrumented methods"""
+    deprecate("The yaaredis module is deprecated and will be deleted.", message="", category=DDTraceDeprecationWarning)
```

Comment:
Let's make the prefix you have here the message argument and say why we're deleting it. 

The prefix arg can just be `"The yaaredis integration is deprecated."` . 

Let's also add a removal_version of "3.0.0"